src/mixins/view.js

```diff
@@ -248,6 +248,6 @@ const ViewMixin = {
   }
 };
 
-_.extend(ViewMixin, BehaviorsMixin, CommonMixin, DelegateEntityEventsMixin, TriggersMixin, UIMixin);
+_.extend(ViewMixin, domMixin, BehaviorsMixin, CommonMixin, DelegateEntityEventsMixin, TriggersMixin, UIMixin);
```

Comment:
The last nit-pick I just noticed.  All of the other Mixins have the first letter uppercase.  Can we do that here too and if there needs to be a stylistic change, do it afterwards?